src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model.mdx

```diff
@@ -144,13 +144,14 @@ The user model is independent of your pricing plan. For how user model relates t
 
 ### Feature impacts of user model [#limitations]
 
-The new user model offers [many benefits](https://newrelic.com/blog/how-to-relic/user-management-and-access-security). Here are some feature impacts of being on the New Relic One user model that may be surprising or unintuitive to our existing customers:
+The new user model offers [many benefits](https://newrelic.com/blog/how-to-relic/user-management-and-access-security). Here are some feature impacts of being on the New Relic One user model that may be not obvious to customers accustomed to our previous user model:
```

Comment:
Do we refer to users as customers? I don't think this is a big deal, but reading "customers" threw me off a bit.